### PR TITLE
Information to users located on the US cluster

### PR DESCRIPTION
**Pull request by ORiborg**

### CODE REVIEW

File: source/includes/Searching/Full/_intro.md [hunk 1]
```diff
@@ -18,9 +18,19 @@ curl
 ```
 > Go [here](#full-searches_request-table) to se a full example of a request body and [here](#full-searches_response) to se a response body.
 
+<h5>EU CLUSTER:</h5>
```
Comment (thedeedawg) [inline source/includes/Searching/Full/_intro.md]: The whole EU/US cluster thing is a very internal thing, and not something our customers are aware of and nor should they be. Instead I would say that we should just word it as customers being in either the US or the rest of the world.

Comment (thedeedawg) [inline source/includes/Searching/Full/_intro.md]: The whole EU/US cluster thing is a very internal thing, and not something our customers are aware of and nor should they be. Instead I would say that we should just word it as customers being in either the US or the rest of the world.

File: source/includes/Searching/Full/_intro.md [hunk 2]
```diff
@@ -18,9 +18,19 @@ curl
...
 
 `POST https://api.cludo.com/api/v3/{CustomerId}/{EngineId}/search`
 
+<h5>US CLUSTER:</h5>
+
+`POST https://api-us1.cludo.com/api/v3/{CustomerId}/{EngineId}/search`
+
+<aside class="warning">
+If you are located on the US cluster <b>ALL</b> your HTTP requests should start with `https://api-us1.cludo.com/api/v3/`. Feel free to <a href="https://www.cludo.com/contact/">contact us</a> if you have any questions!
```
Comment (thedeedawg) [inline source/includes/Searching/Full/_intro.md]: We should only be using raw HTML when it's necessary, `**ALL**`. Also as it's already being emphasized, it's unnecessary to write it in CAPS as well. The same goes for the link `[contact us](https://www.cludo.com/contact/)` - but with that said we should probably add `target="_blank"` to it (which can only be achieved with raw HTML) to not take people away from the docs.

**Comment by ORiborg** [comment]
I will close this one, and create a new pull request

**Pull request closed without merging**